bg_simulator.py

```diff
@@ -58,7 +55,7 @@ def assign_carbs(sugar, last_carbs, sugar_in_range):
         carbs = random.triangular(-50, 100, 60)
     return carbs
 
-def stitch_func(num_days=180):
+def simulate(num_days=10):
```

Comment:
So, now you changed it here, but it was still hard-coded somewhere else before (because I wasn't getting 180 days), and I don't see any other changes. Please remove the hard-coding elsewhere.